packages/techdocs-common/__mocks__/@aws-sdk/client-s3.ts

```diff
@@ -67,20 +57,9 @@ export class S3 {
     });
   }
 
-  upload({ Key }: { Key: string }) {
-    return {
-      promise: () =>
-        new Promise((resolve, reject) => {
-          if (!fs.existsSync(Key)) {
-            reject('');
-          } else {
-            resolve('');
-          }
-        }),
-    };
+  putObject() {
+    return new Promise(resolve => {
+      resolve('');
+    });
   }
```

Comment:
All of these could heavily benefit from `async`/`await`.

```ts
async putObject() {
  return '';
}
```

etc.

But not gonna block on that :) Just a readability thing